queries/nix.scm

```diff
@@ -255,76 +255,16 @@
 ;;!  ^^^^^^
 ;;!  xxxxxxx
 ;;!  ----------------
-(apply_expression
-  [
-    (apply_expression
-      function: (variable_expression
-        name: (identifier) @functionCallee
-      )
-    )
-    (apply_expression
-      [
-        (apply_expression
-          function: (variable_expression
-            name: (identifier) @functionCallee
-          )
-        )
-        (apply_expression
-          [
-            (apply_expression
-              function: (variable_expression
-                name: (identifier) @functionCallee
-              )
-            )
-            (apply_expression
-              (apply_expression
-                function: (variable_expression
-                  name: (identifier) @functionCallee
-                )
-              )
-            )
-          ]
-        )
-      ]
-    )
-  ]
-) @functionCall @_.domain
+(
+  (apply_expression
+    function: (_) @functionCallee
+  ) @functionCallee.domain @functionCall
+  (#not-type? @functionCallee apply_expression)
+)
```

Comment:
This matches the innermost function call, which is why we use the `not-type` here